src/Raven.Studio/typescript/models/resources/info/databaseInfo.ts

```diff
@@ -35,6 +36,16 @@ class databaseInfo extends resourceInfo {
     asResource(): database {
         return new database(this.name, this.isAdmin(), this.disabled(), this.bundles());
     }
+
+    update(databaseInfo: Raven.Client.Data.DatabaseInfo): void {
+        this.updateCurrentInstance(databaseInfo);
+        
+        this.rejectClients(databaseInfo.RejectClients);
+        this.indexingStatus(databaseInfo.IndexingStatus);
+        this.indexingEnabled(databaseInfo.IndexingStatus === ("Running" as Raven.Client.Data.Indexes.IndexRunningStatus));
```

Comment:
Now we don't need to cast running as Raven.Client.Data.Indexes.IndexRunningStatus, because TS can detect this and fail the compilation if string won't match IndexingStatus. 

So: this.indexingEnabled(databaseInfo.IndexingStatus === "Running");